packages/dina-ui/dev.Caddyfile

```diff
@@ -81,4 +81,12 @@ route /api/search-api/* {
 route /api/dina-export-api/* {
     uri replace /api/dina-export-api /api/v1
     reverse_proxy {$REPORT_LABEL_API_ADDRESS}
+}
+
+# Proxy for TGN
+route /tgn/* {
+	uri replace /tgn /TGNService.asmx
+	reverse_proxy http://vocabsservices.getty.edu {
```

Comment:
Reverse proxy here as http vs https causes security issues at AAFC. Cannot merge this pull request until we have a web service that's based on https.

Reply:
Thank you for your feedback. The main issue is that the TGN API does not provide a https endpoint (which leads to a mixed content problem). Christian had the alternative idea to handle the proxy via traefik. In that case, the tgn integration would rely proxy settings in traefik and would not work without traefik.